features/admin.console-settings.v1/components/console-settings-tabs.tsx

```diff
@@ -80,9 +80,30 @@ const ConsoleSettingsTabs: FunctionComponent<ConsoleSettingsTabsInterface> = (
 
     const { t } = useTranslation();
 
-    const { isSubOrganization } = useGetCurrentOrganizationType();
-    const administratorsFeatureConfig: FeatureAccessConfigInterface = useSelector(
-        (state: AppState) => state?.config?.ui?.features?.administrators);
+    const { isFirstLevelOrganization, isSubOrganization, isSuperOrganization } = useGetCurrentOrganizationType();
+
+    const consoleSettingsFeatureConfig: FeatureAccessConfigInterface = useSelector(
+        (state: AppState) => state?.config?.ui?.features?.consoleSettings);
+    const isLoginFlowConfigurationEnabledForFirstLevelOrgs: boolean =
+        !consoleSettingsFeatureConfig?.disabledFeatures?.includes(
+            "consoleSettings.firstLevelOrgloginFlowConfiguration"
+        );
+
+    const isLoginFlowEnabled = () => {
```

Comment:
Shall we check the perf impact of this function and see whether we can benefit from a `useMemo`

Reply:
Fixed with [d587fe7](https://github.com/wso2/identity-apps/pull/6938/commits/d587fe7b7259c4a7ddb0ce2652dc1c38c1278380)